Project.toml

```
@@ -1,7 +1,7 @@
 name = "T8code"
 uuid = "d0cc0030-9a40-4274-8435-baadcfd54fa1"
 authors = ["Johannes Markert <johannes.markert@dlr.de>"]
-version = "0.7.0"
+version = "0.8.0"
```

```suggestion
version = "0.7.1"
```
Is this breaking? As fas as I can see, it only adds new functionality.

---

I adhered to the semantic versioning convention:

https://semver.org/ says:
```
Given a version number MAJOR.MINOR.PATCH, increment the:

    MAJOR version when you make incompatible API changes
    MINOR version when you add functionality in a backward compatible manner
    PATCH version when you make backward compatible bug fixes
```

---

MAJOR is still a zero meaning the whole package is in development stage.

---

As far as I know, if MAJOR is still zero adding functionality in a backward compatible manner can also be done in a PATCH version number. E.g., in Trixi.jl we do that. The MINOR versions is then only bumped for breaking release. Also rules 6. - 8. at [semver.org](https://semver.org/) are only valid for versions vx.y.z with x > 0.

---

Interesting. Then I'll stick to the Trixi convention. No hard feelings on that. Thanks for the info!